neurom/analysis/morphtree.py

```diff
@@ -222,8 +222,8 @@ def partition(tree):
     '''
     def partition_at_point(bif_point):
         '''Partition at each bif point.'''
-        n = n_sections(bif_point.children[0])
-        m = n_sections(bif_point.children[1])
+        n = float(n_sections(bif_point.children[0]))
```

Comment:
Is there a test for this issue? If not, you should add one. Anything "fixed" needs a test that would fail without the fix.